package.json

```diff
@@ -49,8 +49,9 @@
     "cspell": "8.6.0",
     "cz-conventional-changelog": "3.3.0",
     "detect-indent": "7.0.1",
-    "esbuild": "0.14.2",
+    "esbuild": "0.20.1",
```

Comment:
0.20.2 was published a few days ago, so we might as well upgrade to that version

https://github.com/evanw/esbuild/releases/tag/v0.20.2

Reply:
Good catch !